api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -48,7 +48,8 @@ public interface WorkspaceMapper {
   @Mapping(target = "displayName", source = "dbWorkspace.name")
   @Mapping(target = "terraName", source = "fcWorkspace.name")
   @Mapping(target = "googleBucketName", source = "fcWorkspace.bucketName")
-  @Mapping(target = "creator", source = "dbWorkspace.creator.username")
+  @Mapping(target = "creator.userName", source = "dbWorkspace.creator.username")
+  @Mapping(target = "creator.email", ignore = true) // need to work with security before exposing
```

Comment:
when we do get this approval, I want to use a new field like contactEmail or institutionalEmail, to be explicit about which email it is

Reply:
Good call, I added a comment.